test/array.spec.ts

```diff
@@ -5,8 +5,7 @@ const testFile: string = "array";
 
 test('it should create an array', () => {
     const instance: WebAssembly.Instance = getWasmInstanceSync(
-        path.join(__dirname, `${testFile}.tbs`),
-        path.join(__dirname, `bin/${testFile}.wasm`)
+        path.join(__dirname, `${testFile}.tbs`)
```

Comment:
I brought in the change and tried "./array.tbs" but got "Cannot read from ./array.tbs" :( I'd still like to simplify this and have getWasmInstanceSync figure it out, but that's just me, so for now as-is is just fine.